README.md

```diff
@@ -210,4 +210,20 @@ Hue is given in degrees (0-360) and saturation and value in percentage.
 print(bulb.hsv)
 if bulb.is_color:
    bulb.hsv = (180, 100, 100) # set to cyan
+```
+
+## Development Setup
+
+### Docker
+
+Install Docker for your system [Docker for Mac](https://docs.docker.com/docker-for-mac/), [Docker for Windows](https://docs.docker.com/docker-for-windows/), [Docker for Linux](https://docs.docker.com/engine/installation/#server)
```

Comment:
Is this part (especially the links) necessary? I would just move the next subchapter here, and mention in the beginning that the system must have a working docker installation.